en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -176,11 +210,42 @@ This section involves setting up the Traffic Manager node(s) and enabling it to
 
 [![Traffic Manager Connections]({{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png)]({{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png)
 
+1.  Open the `<API-M_HOME>/repository/conf/deployment.toml` file in the Traffic Manager node and change as following to point to Key manager nodes.
+
+    ``` toml tab="Key Manager with HA"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-LB-host]/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+        
+    ``` toml tab="Single Key Manager"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-host]:${mgt.transport.https.port}/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+##### Configuring High Availability (HA) for the Traffic Manager.
+
 1.  If you need to configure High Availability (HA) for the Traffic Manager, use a copy of the existing Traffic Manager instance as the second Traffic Manager active instance and configure a load balancer fronting the two Traffic Manager instances.
 
-2.  Mount the `<API-M_HOME>/repository/deployment/server/executionplans` directory of all the Traffic Manager nodes to a shared file system as a content synchronization mechanism in-order to share the throttling policies between Traffic Manager nodes.
+2. Open the `<API-M_HOME>/repository/conf/deployment.toml` file in the Traffic Manager node and add the following configuration to publish events into other node.
```

Comment:
publish events into other node -> publish events to other nodes.

Reply:
what do you mean by other nodes?

Reply:
this was a event duplicate from any one of the node